private-click-measurement.bs

```diff
@@ -25,120 +25,179 @@ Complain About: missing-example-ids yes
 
 1. A user searches for something on `search.example`'s website.
 2. The user is shown an ad for a product and clicks it.
-3. The ad click source, `search.example`, informs the user agent (see [[#linkformat]]):
-    - That it will want ad click attribution for this click.
-    - What the intended [=ad click destination=] is.
-    - What the [=ad campaign id=] is.
-4. The user agent navigates and takes note that the user landed on the intended [=ad click destination=].
-5. The user's activity on the [=ad click destination=] leads to a [=conversion=].
-6. A third-party HTTP request is made on the ad click destination website to `​https://search.example/.well-known/ad-click-attribution`
-7. The user agent checks for pending ad click attributions for the [=ad click source-destination pair=] and if there's a hit, makes or schedules an HTTP request to `​https://search.example/.well-known/ad-click-attribution` with the [=ad attribution data=].
+3. The [=click source website=], `search.example`, informs the user agent (see [[#linkformat]]):
+    - That it will want attribution for this click.
+    - What the intended [=attribute-on website=] is.
+    - What the [=source id=] is.
+4. The user agent navigates and takes note that the user landed on the intended [=attribute-on website=].
+5. The user's activity on the [=attribute-on website=] leads to a [=triggering event=].
+6. A third-party HTTP request is made on the [=click source website=] to `​https://search.example/.well-known/private-click-measurement/trigger-attribution/` with [=trigger data=].
+7. The user agent checks for stored clicks for the [=click source-attribute-on pair=] and if there's a hit, makes or schedules an HTTP POST request to `​https://search.example/.well-known/private-click-measurement/report-attribution/` with the corresponding [=attribution report=].
 
-    ISSUE: One thing to consider here is whether there should be an option to send the [=ad attribution data=] to the [=ad click destination=] too.
+    ISSUE: One thing to consider here is whether there should be an option to send the [=attribution report=] to the [=attribute-on website=] too.
 
-# Ad Click Source Link Format # {#linkformat}
+# Click Source Link Format # {#linkformat}
 
-This specification adds two attributes to the {{HTMLAnchorElement}} interface. Authors can use these attributes in HTML content like so (where `17` is an [=ad campaign id=] and `https://destination.example/` is an [=ad click destination=]):
+This specification adds two attributes to the {{HTMLAnchorElement}} interface. Authors can use these attributes in HTML content like so (where `17` is an [=source id=] and `https://destination.example/` is an [=attribute-on website=]):
 
 <xmp class="highlight" highlight=html>
-<a adcampaignid="17" addestination="https://destination.example/">
+<a sourceid="17" attributeon="https://destination.example/">
```

Comment:
Some thoughts on these new names:
* I liked that the previous ones had a common prefix, so you could tell they were related.
* `sourceid`, in particular, seems weird if you don't know anything about PCM. It would be fair for someone to wonder if/assume it's the ID of this element in some source document, which for some reason got moved into this attribute from `id=""` by some tool.

I'd like @annevk's or @domenic's thoughts on these attribute name renames.

Reply:
My mistake on `sourceid`. It should be `attributionsourceid`. That's what's implemented in WebKit.

`attributeon` has been cringe-worthy all along but the Google folks wanted it to be "attribute-on" so it's called "the attribute-on website":
https://github.com/WICG/conversion-measurement-api/issues/57#issuecomment-700912187
Referenced in the PCM issue on naming:
https://github.com/privacycg/private-click-measurement/issues/56

Reply:
We'd be happy to re-engage on those issues if a different naming is more appropriate. My apologies if it came across as though our mind was made up on the matter, that wasn't the intention.

Given the benefits of having a common prefix raised above and using "source" for the click side, `attributiondestination` may be a better fit (which was the original name on #56). "attributionsite" is another name that solves the above issues and mixes well with prose.

Reply:
Changing `attributeon` would likely be a breaking change in WebKit. We've been in beta for five weeks with `attributeon`.

Reply:
I like `attributiondestination` much better than `attributeon`. Would Google Chrome be willing to change to that?

Reply:
Yes, we're supportive of the change. I will file a PR against the Conversion Measurement API repo to use "attributiondestination".

cc @csharrison @maudnals

Reply:
Okay, I think you addressed both of my concerns in 8623b5d. Thanks!